### PR TITLE
Nerfs secborgs

### PR DESCRIPTION
**Pull request by Markolie**
**What does this PR do:**
Please see https://github.com/ParadiseSS13/Paradise/pull/11432 for the arguments behind these changes. This is an alternative to that pull request.

This alternative removes the disabler from the security module (and removes the disabler cooldown upgrade from science, as it's no longer useful), slows down the security module and prevents it from being upgraded with VTEC. It also limits the number of security borgs to 1.

**Changelog:**
:cl: Markolie
balance: The cyborg security module no longer has a disabler. The disabler cooldown upgrade can no longer be produced.
balance: The cyborg security module has been slowed down.
balance: The cyborg security module can no longer be upgraded with VTEC.
balance: There can only be one active security borg on the station.
/:cl:

### CODE REVIEW

**Comment by Mitchs98** [comment]
While I'm against both PR's, secborgs are slow enough as they are without a VTEC. If you're removing their disabler, you may as well remove them altogether. As I said on the other PR, their armor is paper, if they're having to move into baton range while slow as dirt they may as well not exist. 

I feel that's the reason this PR was made in all honesty, or it seems that way. Make them next to useless to the point they may as well be removed so people will just fold and favor the removal PR.

**Comment by Kyep** [comment]
I'm against this too, though not as strongly as the other PR.
This PR does fit with my vision for sec borgs (strong but slow heavy artillery).
However, it doesn't actually address the claimed problem with sec borgs, which is that many of them can be made, to the point you have a half dozen of them running around. At that point I don't think these nerfs will actually shift the balance much.

If we want to nerf sec borgs. If. If we want to nerf them, the most logical thing I can suggest would be simply capping their numbers. Limit them to 2-3 sec borgs on station at once. That way it is no longer possible to field an army of them, and they remain the big guns of the sec force that trundle their way towards whatever the threat is, keeping antags on the run rather than letting them camp areas for long periods of time.

**Comment by Mitchs98** [comment]
> I'm against this too, though not as strongly as the other PR.
> This PR does fit with my vision for sec borgs (strong but slow heavy artillery).
> However, it doesn't actually address the claimed problem with sec borgs, which is that many of them can be made, to the point you have a half dozen of them running around. At that point I don't think these nerfs will actually shift the balance much.
> 
> If we want to nerf sec borgs. If. If we want to nerf them, the most logical thing I can suggest would be simply capping their numbers. Limit them to 2-3 sec borgs on station at once. That way it is no longer possible to field an army of them, and they remain the big guns of the sec force that trundle their way towards whatever the threat is, keeping antags on the run rather than letting them camp areas for long periods of time.

This is something I'd be in favor of. A PR that puts a global cap on security borgs, rather than wholesale nerfing them or removing them altogether.

**Comment by Markolie** [comment]
Sadly enough I do not think that resolves the issue that I see with security borgs, namely that a security borg paired up with an officer is an unstoppable combination. Yes, the issue with security borgs is magnified significantly if there's more security borgs on station, but in my opinion even one competent security borg causes plenty of issues.

And mind you, none of these nerfs are set in stone.

**Comment by Kyep** [comment]
Frankly, I wouldn't treat security borgs any differently from regular borgs. 
Cap alll types of borg to 3. Having 4 medical borgs is a waste of resources, just as having 4 security borgs is arguably overkill.

**Comment by PidgeyThePirate** [comment]
I generally agree with the sentiment that being in range of a secborg should not be somewhere an antagonist would want to be. Going hand to hand with a secborg should be suicidal, at least without preparation. I'd also be for a shifting cap at one of each module (apart from service) per 30 people or so. But if you do this then you need to make secborgs less papery so they can still flush out entrenched antags.

**Comment by Shockpoint** [comment]
I disagree with this PR as well, but not as strongly as outright removal does not fix anything. I'll get around to my points in a bit as I'm busy atm.

**Comment by Couls** [comment]
> This PR does fit with my vision for sec borgs (strong but slow heavy artillery).
If we really want secborgs to be slow and heavy artillery why can they get vtec modules that make them just as fast as an officer with no downside? not to mention most times the sec borg is the first responder to crimes because of access restrictions given to officers

**Comment by Tayyyyyyy** [comment]
Doesn't this leave sec borgs with no way to get around in zero g?

**Comment by Markolie** [comment]
Negative, that’s the ion thruster upgrade.

**Comment by davidchan13** [comment]
This makes no sense. Why would you take away the range weapon while simaltanously nerfing speed. The sec Borg isn't intended to tank spiders and deal lethal damage, it's intended to chase down greytide.

**Comment by Sihsse** [comment]
Just a bit of a spitball here. Rather than removing the disabler option why not increase the amount of energy is used for the Borg to fire it, making it the more expensive option between the baton and the ranged weapon.

It'll allow the borg the ability to still use it however they'll have to be smarter in their application of it or risk drying their battery out in a firefight, similarly the cool down upgrade only means they'll drain themselves faster with the use, so the disabler option becomes more situational instead of the weapon of choice when apprehending.

**Comment by Emanthealmighty** [comment]
Suggestion: someone mentioned that using flashes against borgs can prove to be hard, what if you removed the stun baton but kept the disabler? Then the sec borg would really be slow artillery, as well as removing its ability to spam stun traitors along with the rest of sec. (So if you have two borgs and one sec, only the human security officer will be able to properly stun you to keep you down, giving you a chance to do all sort of things)

**Comment by Markolie** [comment]
I'm okay with the inverse, where they keep their disabler and lose their baton.

**Comment by Mitchs98** [comment]
I'm honestly against removing any tools from them at all, I don't see the reason to, but if anything you can at-least use disabler to keep people down once you down them..

**Comment by MRgaty** [comment]
I think limiting the number of sec borgs is a good idea, still just having 1 sec borg independent of the round population is almost like having no borgs.

**Comment by taukausanake** [comment]
I feel more than one secborg people are going to complain still. One should be enough so long as there are enough security members. Maybe secborgs can scale inversely with security or something.

I just realized that the Roboticist's job might be a lot less meaning full now

**Comment by Imotepchief** [comment]
I posted on the other linked topic, but I'll reiterate here.
Sec borgs don't need altering, they are fine **and fairly balanced**, as they are.
Removing their ranged capability makes them effectively useless.
There are many easy counters to them when they are alone, and are often unable to be used effectively with a partner, unless both individuals excel at their job.

Any real gripes that you may be feeling towards sec borgs, is likely down to a couple of things.
Either a number of low quality "tide" players taking borg roles and running sec, thus causing issues.
Or
Antags thinking they can go about things haphazardly and still achieve their objectives.

**Comment by Couls** [comment]
Lets bring back peacekeepers!

**Comment by MrMagolor** [comment]
>The only problem with this would be that sec borgs would have a hard time cuffing and securing one guy by themselves.

Perhaps the point is that they _shouldn't_ be able to do that?

**Comment by Mitchs98** [comment]
> > The only problem with this would be that sec borgs would have a hard time cuffing and securing one guy by themselves.
> 
> Perhaps the point is that they _shouldn't_ be able to do that?

That'd make them pretty useless if they couldn't. They shouldn't _always_ have to have a partner to be able to do their basic function.

**Comment by The-Driker** [comment]
> This PR does fit with my vision for sec borgs (strong but slow heavy artillery).

With all due respect, you can't call something that only has melee as a weapon, artillery. 

> I feel that's the reason this PR was made in all honesty, or it seems that way. Make them next to useless to the point they may as well be removed so people will just fold and favor the removal PR.

I feel I must agree.
I want to apologize now as this is going to be long-winded and preface this with the fact that while I do in fact main as a cyborg/android. I personally don't care for security cyborgs and only play it when told to. Playing cyborgs is thankless as is and people typically wont be punished for killing you for arbitrary reasons since you have less rights as a player, and playing the security module is suffering. 

I've given it a good deal of thought and I honestly can't coherently come up with a reason to play as a security cyborg that is even slower than normal, incapable of dealing with people at range and made of tissue paper [because cyborgs very much are]. It would literally just be the standard module, but even worse, and that only gets picked when there's a rogue AI or you expect to get illegals for an energy sword. 

I've been playing this game a long time and from what I've experienced, security cyborgs fill a very specific gap. A gap that other servers with higher skill ceilings typically don't have to worry about as much. Simply put, they force people to consider the consequences of being overly aggressive too early. You can't disarm spam a cyborg and steal it's weapon and headset in the middle of the open hallways, which is enough to make some people despise them. When they are around, they force people to go “oh that's something I have to do some minor prep-work to deal with before I go ham”, where as regular officers are easily dealt with using basic gear or your own hands. I could probably make TG comparisons about traitor AIs and borgs, but their AIs have an expanded tool-set and can do objectives without borgs. 

**TL;DR Servers full of people who power-game constantly and have mostly skilled players don't need something like security borgs, because random assistants are typically capable of stopping overly aggressive antags. Paradise isn't a place that encourages those types of players. if I HAD to pick between nerfing or removal, I'd pick removal and also remove traitor AIs.**

**Comment by shazbot194** [comment]
Shouldn’t this also have a controversial tag?

**Comment by Kyep** [comment]
To clarify my earlier comment:
When I said "_If we want to nerf them, the most logical thing I can suggest would be simply capping their numbers. Limit them to 2-3 sec borgs on station at once._" I meant that instead of, not in addition to, the earlier nerfs in the PR. Since the earlier nerfs are still in this PR, I cannot support it.

**Comment by Kyep** [comment]
I assume this is going to be rejected: while it has a maint in favor of it, it has no head in favor of it, and it has two objections.

**Comment by variableundefined** [comment]
Closing since it has two heads rejection
